sql/core/src/main/scala/org/apache/spark/sql/KeyValueGroupedDataset.scala

```diff
@@ -21,7 +21,7 @@ import scala.collection.JavaConverters._
 
```

Comment:
`def mapValues` in this file is not updated, shouldn't it use `ScopedExpression`?

Reply:
No, as it is fine to reference the `value` column provided by `mapValue`'s `AppendColumns`. That is what we want. `ScopedExpression` hides unwanted columns, here we do not want to hide `value`.